src/openforms/authentication/contrib/digid/tests/test_signicat_integration.py

```diff
@@ -0,0 +1,608 @@
+
+        config = DigidConfiguration.get_solo()
+        config.certificate = cert
+        # broker insists using https
+        config.base_url = config.entity_id = "https://localhost:8000"
+        # config.authn_requests_signed =  False
+        config.idp_service_entity_id = SIGNICAT_BROKER_BASE / "sp/saml"
+        config.artifact_resolve_content_type = XMLContentTypes.text_xml
+        config.attribute_consuming_service_index = "1"
+        config.service_name = "Test"
+        config.requested_attributes = [{"name": "bsn", "required": True}]
+        config.want_assertions_signed = False
+        config.want_assertions_encrypted = False
+        config.slo = False
+        config.attribute_consuming_service_index = "1"
+
+        with METADATA.open("rb") as md_file:
+            config.idp_metadata_file = File(md_file, METADATA.name)
+            config.save()
+
+    def _get_cassette_library_dir(self):
+        return str(TEST_FILES / "vcr_cassettes" / self.__class__.__qualname__)
+
+    def _get_vcr_kwargs(self):
+        kwargs = super()._get_vcr_kwargs()
+        # https://vcrpy.readthedocs.io/en/latest/usage.html#record-modes
+        kwargs["record_mode"] = "none"  # in CI
+        # kwargs["record_mode"] = "once"  # in dev
+        return kwargs
+
+    def setUp(self):
+        super().setUp()
+
+        clear_caches()
+        self.addCleanup(clear_caches)
+
+        # We're freezing the time to whatever is on the cassette, because parts of the
+        # body of the SAML messages are time dependant. (e.g. expiration datetimes)
+        #
+        # (this is funny if you're old enough to have seen your VCR with a blinking time
+        # and it missed recording episodes during your holiday)
+        if self.cassette.responses:
+            now = self.cassette.responses[0]["headers"]["date"][0]
+            time_ctx = freeze_time(now)
+            self.addCleanup(time_ctx.__exit__)
+            time_ctx.__enter__()
+
+    def test_login_with_too_low_a_loa_fails(self):
+        session: requests.Session = requests.session()
+        form = FormStepFactory.create(
+            form__slug="slurm",
+            form__authentication_backends=[PLUGIN_ID],
+            form__authentication_backend_options={
+                PLUGIN_ID: {"loa": DigiDAssuranceLevels.substantial}
+            },
+            form_definition__login_required=True,
+        ).form
+
+        login_url = reverse(
+            "authentication:start", kwargs={"slug": form.slug, "plugin_id": PLUGIN_ID}
+        )
+        form_path = reverse("core:form-detail", kwargs={"slug": form.slug})
+        form_url = furl("http://testserver/") / form_path
+
+        our_faux_redirect = self.client.get(f"{login_url}?next={form_url}", follow=True)
+        # do the JS submit to get redirected to Signicat broker
+        method, redirect_url, form_values = _parse_form(our_faux_redirect)
+        self.assertTrue(session.request(method, redirect_url, data=form_values).ok)
```

Comment:
I think it'd make more sense to build this test as a `WebTest` from django-webtest than `TestCase` in this case, since it has the mechanisms to do:

```
login_page = self.app.get(login_url, {"next": str(form_url)})
hidden_form = login_page.form
redirect = hidden_form.submit()
...
```

It's kind of a middle ground between Django's test client and full blown E2E tests.